std/uni/package.d

```diff
@@ -9834,25 +9834,29 @@ dchar toLower(dchar c)
     Returns:
         An array with the same element type as `s`.
 +/
-ElementEncodingType!S[] toLower(S)(S s)
-if (isSomeString!S || (isRandomAccessRange!S && hasLength!S && hasSlicing!S && isSomeChar!(ElementType!S)))
+ElementEncodingType!S[] toLower(S)(return scope S s) @trusted
```

Comment:
Can you explain why you added `@trusted`?

Reply:
It's from the `@trusted` lambda in the original code.

Reply:
I didn't look at it correctly 🙃